content/_changelogs/9.2.0.md

```diff
@@ -0,0 +1,75 @@
+  memory crashes on certain browsers. Fixes
+  [#18549](https://github.com/cypress-io/cypress/issues/18549).
+- Removed automatic retries for failed HTTP requests through the proxy. This
+  fixes an issue where failed requests could be re-sent too many times in some
+  conditions. This change could increase the number of failed requests that your
+  app sees. Fixes [#19043](https://github.com/cypress-io/cypress/issues/19043).
+- Reduced the occurrence of an issue where logs for `fetch` and `xhr` requests
+  could be associated with the wrong request. Fixes
+  [#19043](https://github.com/cypress-io/cypress/issues/19043).
+- Tests that are skipped within `then` blocks will no longer throw errors
+  causing the test to fail. Tests that are skipped outside of `then` blocks will
+  no longer trigger the fail event. This will prevent screenshots from happening
+  from errors thrown by the fail event.Fixes
+  [#14867](https://github.com/cypress-io/cypress/issues/14867) and
+  [#17660](https://github.com/cypress-io/cypress/issues/17660).
+- Fixed a regression in [9.0.0](/guides/references/changelog#9-0-0) where a
+  fixture provided in a static response to `cy.intercept()` did not support
+  passing `null` to encoding to read the fixture as a Buffer. This identified an
+  undocumented 9.0.0 Breaking Change where the default read behavior of a
+  fixture changed from a Buffer to being read with `utf8` encoding. Fixes
+  [#19344](https://github.com/cypress-io/cypress/issues/19344).
+- Fixed a regression in [9.0.0](/guides/references/changelog#9-0-0) where
+  `cy.contains()` attempted to ignore `<script>` and `<style>` elements found
+  within `<body>`. by deleting them from the dom. This behavior was corrected to
+  ignore the elements without deleting them. Fixes
+  [#19377](https://github.com/cypress-io/cypress/issues/19377).
+- Cypress will no longer crash when proxying an ill formed request. For example,
+  if the application under test has a resource of `"http: //localhost/asset.js"`
+  (notice the extraneous space), Cypress will now log a debug message and the
+  asset will fail to load. Fixes
+  [#9220](https://github.com/cypress-io/cypress/issues/9220).
+- Correct `Cypress.Command.add()` and `Cypress.Command.override()` TypeScript
+  types. Fixes [#18879](https://github.com/cypress-io/cypress/issues/18879),
+  [#19095](https://github.com/cypress-io/cypress/issues/19095) and
+  [#18940](https://github.com/cypress-io/cypress/issues/18940).
+  - Custom command implementations typings take into account `prevSubject`
+    variants.
+  - Custom command implementations now allows to NOT return a value.
+  - Custom command overwrites typings take into account `originalFn` function.
+- Add types for `Cypress.session.clearAllSavedSessions()`. Fixes
+  [#19363](https://github.com/cypress-io/cypress/issues/19363).
+
+**Dependencies:**
+
+- Upgraded `ssri` from `6.0.2` to `6.0.2` to mitigate
```

Comment:
Should be `6.0.1` to `6.0.2`

Reply:
[`27d3cd4` (#4270)](https://github.com/cypress-io/cypress-documentation/pull/4270/commits/27d3cd48adbbee1878b30255c44a988f82a9f809)